src/main/scala/io/snyk/plugin/ui/annotators/SnykMavenRedUnderlineAnnotator.scala

```diff
@@ -0,0 +1,41 @@
+package io.snyk.plugin.ui.annotators
+
+import com.intellij.lang.annotation.{AnnotationHolder, Annotator}
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.psi.PsiElement
+import io.snyk.plugin.IntellijLogging.ScalaLogger
+import io.snyk.plugin.datamodel.SecurityVuln
+import io.snyk.plugin.ui.state.SnykPluginState
+
+/**
+  * Red underline annotator for Maven vulnerabilities.
+  */
+class SnykMavenRedUnderlineAnnotator extends Annotator {
+  protected lazy val log: ScalaLogger = new ScalaLogger(Logger.getInstance(this.getClass))
+
+  override def annotate(element: PsiElement, holder: AnnotationHolder): Unit = {
+    val pluginState = SnykPluginState.forIntelliJ(element.getProject)
+
+    val latestScanForSelectedProject = pluginState.latestScanForSelectedProject
+
+    if (latestScanForSelectedProject.isEmpty) {
+      return
+    }
+
+    latestScanForSelectedProject.get.vulnerabilities.seq.foreach(vulnerability => {
+      val securityVulnerability = vulnerability.asInstanceOf[SecurityVuln]
+
+      val vulnerabilityName = securityVulnerability.name
+      val vulnerabilityNameParts = vulnerabilityName.split(":")
+
+      vulnerabilityNameParts.foreach(name => {
+        if (element.getText == name) {
+          import com.intellij.openapi.util.TextRange
+
+          val range = new TextRange(element.getTextRange.getStartOffset, element.getTextRange.getEndOffset)
+          holder.createErrorAnnotation(range, "Vulnerability issue")
```

Comment:
Where do I see this error text?

Reply:
When a user hovers over dependency text.

<img width="1395" alt="Screen Shot 2020-01-28 at 4 19 43 PM" src="https://user-images.githubusercontent.com/7049329/73271960-383e9680-41ea-11ea-8d4e-6157e672fec7.png">

I think we can add a more useful description. For example, in this case, show "XML External Entity (XXE) Injection" message.

Reply:
Maybe then `Vulnerable package`?

Reply:
Ok. I'll change it.